tests/CMakeLists.txt

```diff
@@ -92,17 +92,19 @@ link_libraries(${DETOURS_LIBRARY} Catch2::Catch2WithMain)
 
 add_subdirectory(app)
 add_subdirectory(cpplatest)
+add_subdirectory(cppwinrt-com-server)
+add_subdirectory(cppwinrt-com-server-custom-module-lock)

```

Comment:
You should modify `scripts/runtests.cmd` to ensure that these tests run locally for people and on CI machines.

Reply:
Updated. Ran `runtests.cmd` locally and all `cppwinrt-com-server*` passed.